tests/utils.go

```diff
@@ -74,13 +77,20 @@ var KubeVirtOcPath = ""
 var KubeVirtVirtctlPath = ""
 var KubeVirtInstallNamespace string
 
+var CurrentClusterType = NewClusterTypeValue()
+var DeployTestingInfrastructureFlag = false
+var PathToTestingInfrastrucureManifests = ""
+
 func init() {
-	flag.StringVar(&KubeVirtVersionTag, "tag", "latest", "Set the image tag or digest to use")
-	flag.StringVar(&KubeVirtRepoPrefix, "prefix", "kubevirt", "Set the repository prefix for all images")
+	flag.StringVar(&KubeVirtVersionTag, "docker-tag", "latest", "Set the image tag or digest to use")
+	flag.StringVar(&KubeVirtRepoPrefix, "docker-prefix", "kubevirt", "Set the repository prefix for all images")
 	flag.StringVar(&KubeVirtKubectlPath, "kubectl-path", "", "Set path to kubectl binary")
 	flag.StringVar(&KubeVirtOcPath, "oc-path", "", "Set path to oc binary")
 	flag.StringVar(&KubeVirtVirtctlPath, "virtctl-path", "", "Set path to virtctl binary")
 	flag.StringVar(&KubeVirtInstallNamespace, "installed-namespace", "kubevirt", "Set the namespace KubeVirt is installed in")
+	flag.Var(&CurrentClusterType, "cluster-type", "Set type of cluster (k8s, ocp)")
```

Comment:
Something like this should do the job (instead of setting it from outside):

```golang
func IsOpenShift() bool {
	virtClient, err := kubecli.GetKubevirtClient()
	PanicOnError(err)

	result := virtClient.RestClient().Get().AbsPath("/version/openshift").Do()
        
        // TODO the if error is not a 404 error, we have a communication error, this means that
       // the detection failed
	if result.Error() == nil {
		return true
	}
        return false
}
```